src/infi/systray/traybar.py

```diff
@@ -166,13 +196,10 @@ def _destroy(self, hwnd, msg, wparam, lparam):
         nid = NotifyData(self._hwnd, 0)
         Shell_NotifyIcon(NIM_DELETE, ctypes.byref(nid))
         PostQuitMessage(0)  # Terminate the app.
-        # TODO * release self._menu with DestroyMenu and reset the memeber
-        #      * release self._hicon with DestoryIcon and reset the member
-        #      * release loaded menu icons (loaded in _load_menu_icon) with DeleteObject
-        #        (we don't keep those objects anywhere now)

```

Comment:
Is this TODO not necessary any more? I think we're still not releasing these resources.